test/plugins/validation/2and3/schema-ibm.js

```diff
@@ -1328,4 +1328,32 @@ describe('validation plugin - semantic - schema-ibm - OpenAPI 3', () => {
       'Enum values must follow case convention: lower_snake_case'
     );
   });
+
+  it('should skip validation for non string values', () => {
+    const config = {
+      schemas: {
+        snake_case_only: 'warning'
+      }
+    };
+
+    const spec = {
+      definitions: {
+        Thing: {
+          type: 'object',
+          description: 'thing',
+          properties: {
+            color: {
+              type: 'string',
+              description: 'some color',
+              enum: [1, 2, 3]
```

Comment:
Will you update this property so it makes more sense? You describe a string property called "color", but the enums are numbers. It's a good test case, we just want the setup to be logical

Reply:
The `type` is still "string", change it to "integer"

Reply:
And actually, if its a "list of integers", it should probably be an array. So maybe change it to "integer"